staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -927,6 +927,10 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume `json:"allowedFlexVolumes,omitempty" protobuf:"bytes,18,rep,name=allowedFlexVolumes"`
+	// AllowedCSIDrivers is a whitelist of allowed CSI drivers.  This slice will be checked
+	// to verify which CSI driver is allowed to run in the cluster.
+	// +optional
+	AllowedCSIDrivers []AllowedCSIDriver `json:"allowedCSIDrivers,omitempty" protobuf:"bytes,23,rep,name=allowedCSIDrivers"`
```

Comment:
What does this have to do with inline volumes?

Reply:
Pod Security Policy can be used to enable/disable CSI drivers.

See https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190122-csi-inline-volumes.md#specifying-allowed-inline-drivers-with-podSecuritypolicy